Makefile

```diff
@@ -31,6 +33,13 @@ ifeq ($(PARENT_BRANCH_SUFFIX), main)
 CERT_MANAGER_OPERATOR_BRANCH = master
 endif
 
+## current branch name of the istio-csr submodule.
```

Comment:
nit: [non-blocking review] another approach would be to use if-else block for setting the branch variables to change readbility to "conditional set A or B" vs "set A, conditional override to B".